paymentsheet/src/test/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethodTest.kt

```diff
@@ -87,11 +88,6 @@ class SupportedPaymentMethodTest {
 
                         csvOutput.append(actualLine)
 
-                        PaymentIntentTestCase(
-                            testInput,
-                            testOutput
-                        )
```

Comment:
This was the only usage of `PaymentIntentTestCase`, so I figure we can get rid of it.